skimage/data/tests/test_data.py

```diff
@@ -7,13 +7,6 @@
 import pytest
 
 
-def test_data_dir():
-    # data_dir should be a directory people can use as a standard directory
-    # https://github.com/scikit-image/scikit-image/pull/3945#issuecomment-498141893
-    data_dir = data.data_dir
-    assert 'astronaut.png' in os.listdir(data_dir)
```

Comment:
This test might fail until `download_all()` was called at least once. I opted to remove it for now and rather think about how we might refactor this stuff to make it also easier to test (e.g. less functions relying on module-level attributes).